app/controllers/api/v1/api_keys_controller.rb

```diff
@@ -47,19 +47,25 @@ def update
   private
 
   def check_mfa(user)
-    if user&.mfa_gem_signin_authorized?(otp)
+    if user&.webauthn_credentials&.any? && otp.blank?
```

Comment:
```suggestion
    if user&.webauthn_credentials.present? && otp.blank?
```

might be a bit clearer?

to clean it up even more, `return false unless user` at the top would get rid of many of the `&.` s

Reply:
Fixed!